msvc/vsimporter-templates/WinStore10-App/App.xaml.h

```diff
@@ -13,16 +13,17 @@ namespace $safeprojectname$ {
 /// </summary>
 ref class App sealed {
 protected:
-    virtual void OnLaunched(Windows::ApplicationModel::Activation::LaunchActivatedEventArgs^ e) override;
-    virtual void OnActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs^ e) override;
+    virtual void OnLaunched(Windows::ApplicationModel::Activation::LaunchActivatedEventArgs ^ e) override;
+    virtual void OnActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs ^ e) override;
 #ifdef ENABLE_BACKGROUND_TASK
-    virtual void OnBackgroundActivated(Windows::ApplicationModel::Activation::BackgroundActivatedEventArgs^ e) override;
+    virtual void OnBackgroundActivated(Windows::ApplicationModel::Activation::BackgroundActivatedEventArgs ^ e) override;
 #endif
+    void OnFileActivated(Windows::ApplicationModel::Activation::FileActivatedEventArgs ^ args) override;
```

Comment:
oops. I fixed it with the .cpp but not .h


---
In reply to: [82850185](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82850185) [](ancestors = 82850185)